File: Hospital/HospitalClassLib/Model/EquipmentTransferEvent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hospital.Model
{
    public class EquipmentTransferEvent
    {
        public int Id { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public int? InitalRoomId { get; set; }
        public Room InitalRoom { get; set; }
        public int? DestinationRoomId { get; set; }
        public Room DestinationRoom { get; set; }
        public int? InventoryItemId { get; set; }
        public InventoryItem InventoryItem { get; set; }
        public int Quantity { get; set; }
    }
}

File: WebClients/ManagerWebClient/src/app/model/equipment-transfer-event.ts
export class EquipmentTransferEvent {
  startDate: Date;
  endDate: Date;
  initialRoomId: number;
  destinationRoomId: number;
  inventoryItemId: number;
  quantity: number;
}

export class EquipmentTransferEventDTO {
  id: number;
  timePeriod: TimePeriod;
  initialRoomInventoryId: number;
  destinationRoomInventoryId: number;
  quantity: number;
}

export class TimePeriod{
  startTime : Date;
  endTime : Date;
}
